website/docs/cloud-docs/users-teams-organizations/permissions.mdx

```diff
@@ -324,6 +328,32 @@ In order to remove a user from the organization, the holder of this permission m
 
 ~> This permission is intended to allow owners of large organizations to delegate membership management to another trusted team, and should be granted to only teams of trusted users. **Assign with caution:** Users with this permission are able to add themselves to any visible team, and inherit the permissions of any visible team.
 
+#### Manage Teams
+
+Allows members to create, update, and delete teams.
+
+This permission grants the ability to update a team names and SSO IDs but not its organization access settings. On its own this permission does not allow users to create, update, delete, or otherwise access secret teams.
+
+This permission confers all of the permissions granted by the manage membership permission.
+
+~> This permission is intended to allow owners of large organizations to delegate team management to another trusted team, and should be granted to only teams of trusted users. **Assign with caution:** Users with this permission are able to update or delete any visible team. Because this permission also confers the manage membership permission these users could also add themselves to any visible team.
+
+#### Manage Organization Access
+
+Allows members to update the organization access settings of teams.
+
+On its own this permission does not allow users to create, update, delete or otherwise access secret teams.
+
+This permission confers all of the permissions granted by the manage teams and manage membership permissions.
+
+~> This permission is intended to allow owners of large organizations to delegate further team management control to another trusted team, and should be granted to only teams of trusted users. **Assign with caution:** Members with this permission will be able to update all organization access settings for any team visible to them.
+
+### Access Secret Teams
```

Comment:
I _think_ this should be "Include secret teams" here, because this cloud-docs section usually maps to the UI vs the API docs